tests/app/db.py

```diff
@@ -567,13 +567,22 @@ def create_api_key(service, key_type=KEY_TYPE_NORMAL, key_name=None):
     return api_key
 
 
-def create_inbound_number(number, provider='mmg', active=True, service_id=None):
+def create_inbound_number(
+    number,
+    provider='mmg',
```

Comment:
Can we please make that aws ses? We don't use `mmg`, seems silly to leave it there.

Reply:
done